EnglishFilter/sections/general_extensions.txt

```diff
@@ -229,8 +229,10 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74115
+deportealdia.live#%#//scriptlet("prevent-window-open")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/73799
-techgeek.digital#%#//scriptlet("prevent-window-open")
+noticiasesports.live,techgeek.digital#%#//scriptlet("prevent-window-open")
```

Comment:
No need. We have the rule in the `AdGuard Spanish`. Please check your changes before commiting to our repo.